index.fr.md

```diff
@@ -0,0 +1,71 @@
+---
+# Translation info https://www.w3.org/wiki/WAI/Website/Translate
+
+title: "Vue d'ensemble : enseignement et promotion"
+nav_title: "Vue d'ensemble"
+
+lang: fr
+last_updated: 2020-12-31
+permalink: /teach-advocate/fr
+
+translator:
+- name: "Sofia Ahmed"
+contributors:
+- name: "Sandra Velarde Gonzalez (ETNIC)"
+
+github:
+  repository: w3c/wai-teach-advocate-overview
+  path: "index.fr.md"
+
+feedbackmail: wai@w3.org
+class: tight-page
+footer: > # Text in footer in HTML
+  <p><strong>Date :</strong> Mise à jour : 12 mai 2020.</p>
+  <p><strong>Rédactrice :</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Développé avec les données provenant du <em>Groupe de travail Éducation et Promotion</em> (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+ref: /teach-advocate/
+---
+
+{::nomarkdown}
+{% include box.html type="start" title="Résumé" class="" %}
+{:/}
+
+Liens vers les ressources liées à l'enseignement de l'accessibilité Web et la promotion en faveur de l'accessibilité Web.
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include toc.html type="start" title="Table des matières" %}
+{:/}
+
+- This will be replaced by an automatically generated TOC when using Markdown formatting.
+{:toc}
+
+{::nomarkdown}
+{% include toc.html type="end" %}
+{:/}
+
+## Ressources d'enseignement {#train}
+
+[[Comment rendre vos présentations accessibles à tous]](/teach-advocate/accessible-presentations/)
+: Vous aide à rendre vos présentations, vos discussions, vos réunions, et vos formations accessibles à l'ensemble de votre public potentiel, y compris les personnes handicapées, entre autres.
+
+[[Programmes d'étude sur l'accessibilité Web : une structure pour créer vos propres cours]](/curricula/)
+: Fournit des programmes d'étude pour vous aider à créer des cours sur l'accessibilité Web pour différents rôles. Ils définissent les objectifs de l'apprentissage et fournissent des idées pour enseigner et évaluer les connaissances. Le programme "Introduction à l'accessibilité Web" est actuellement disponible. Des programmes supplémentaires seront disponibles en 2020 et en 2021.
+
+[[Créer des présentations et des formations sur l'accessibilité Web]](/teach-advocate/accessibility-training/)
+: Fournit des informations pour les orateurs, les conférenciers, les professeurs et autres présentateurs pour aider leur public à mieux comprendre l'accessibilité Web. Vous trouverez différentes informations allant des slides introductives pour les débutants dans l'accessibilité Web, aux statistiques et aux démos pour les formateurs expérimentés. (Note : la majorité de ce contenu date de 2013. La page [Programmes d'étude](/curricula/) renvoie à des ressources plus récentes.)
+
+[Démonstration : avant, après](https://www.w3.org/WAI/demos/bad/)
```

Comment:
(en anglais) is usually generated by our scripts. It didn't for this resource , because it is under a separate system. In this case, I'm confident that we will not have https://www.w3.org/WAI/demos/bad/ translated for a long time, if at all (we still have pending updates to it, and it is not set up for translation).

Therefore, for this one, we could add it manually.